core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -416,6 +411,19 @@ import scala.util.control.NonFatal
   def schedulePollTask(): Unit =
     timers.startSingleTimer(PollTask, pollMsg, settings.pollInterval)
 
+  /**
+   * Sends an extra `Poll` request to self.
+   * Enqueing an extra poll via the actor mailbox allows other requests to be handled
+   * before the actual poll is executed.
+   *  When many requestors, e.g. many partitions with committablePartitionedSource the
+   * performance is much improved by collecting more requests/commits before performing the poll.
+   */
+  private def requestExtraPoll(): Unit =
+    if (!delayedPollInFlight) {
```

Comment:
I think the flag should match the method, so `extraPollInFlight` or `requestDelayedPoll()`.